chapters/ch27.md

```diff
@@ -1,38 +1,37 @@
 
 
 * **Decrease the model size** ​(such as number of neurons/layers): ​*Use with caution*.​ This technique could decrease variance, while possibly increasing bias. However, I don’t recommend this technique for addressing variance. Adding regularization usually gives better classification performance. The advantage of reducing the model size is reducing your computational cost and thus speeding up how quickly you can train models. If speeding up model training is useful, then by all means consider decreasing the model size. But if your goal is to reduce variance, and you are not concerned about the computational cost, consider adding regularization instead.
-->
+-> * ** Giảm kích thước mô hình ** (chẳng hạn như số lượng neurons / tầng): * Sử dụng một cách thận trọng *. Kỹ thuật này có thể làm giảm phương sai, trong khi có thể làm tăng độ chệch. Tuy nhiên, tôi không khuyến khích kỹ thuật này để giải quyết phương sai. Thêm regularization thường cho hiệu suất phân loại tốt hơn. Ưu điểm của việc giảm kích thước mô hình là giảm chi phí tính toán của bạn và do đó tăng tốc độ bạn huấn luyện mô hình. Nếu tăng tốc độ huấn luyện mô hình là hữu ích, thì bằng mọi cách hãy xem xét giảm kích thước mô hình. Nhưng nếu mục tiêu của bạn là giảm phương sai và bạn không quan tâm đến chi phí tính toán, thay vào đó, hãy xem xét việc thêm regularization.
 
 
 Here are two additional tactics, repeated from the previous chapter on addressing bias:
-->
+-> Đây là hai chiến thuật bổ sung, được lặp lại từ chương trước về giải quyết độ chệch:
 
 
 * **Modify input features based on insights from error analysis​**: Say your error analysis inspires you to create additional features that help the algorithm to eliminate a particular category of errors. These new features could help with both bias and variance. In theory, adding more features could increase the variance; but if you find this to be the case, then use regularization, which will usually eliminate the increase in variance.
-->
+-> * ** Thay đổi các đặc trưng đầu vào dựa trên hiểu biết sâu sắc từ phân tích lỗi **: Giả sử rằng việc phân tích lỗi của bạn truyền cảm hứng cho bạn để tạo các đặc trưng bổ sung giúp thuật toán của bạn loại bỏ một hạng mục lỗi cụ thể. Những đặc trưng mới này có thể giúp với cả sự độ chệch và phương sai. Về lý thuyết, việc thêm nhiều đặc trưng có thể làm tăng phương sai; nhưng nếu bạn gặp trường hợp này, hãy sử dụng regularization, thường sẽ loại bỏ sự gia tăng phương sai.
```

Comment:
ờ đây "có thể giúp với" có thể dịch là "giảm" ko bạn? "có thể giúp với cả sự độ chệch và phương sai" --> giảm cả độ lệch và phương sai? (nếu bạn giữ nguyên mình cũng ok)

> hãy sử dụng regularization, thường sẽ loại bỏ sự gia tăng phương sai

mình nghĩ có thể giữ nguyên từ which trong bản gốc, dịch là: "hãy sử dụng regularization, việc này thường sẽ loại bỏ sự gia tăng phương sai". Còn nếu bạn sợ lặp từ "việc" trong câu thì có thể dịch cả câu thế này:
> Về lý thuyết, thêm nhiều đặc trưng có thể làm tăng phương sai; nhưng nếu bạn gặp trường hợp này, hãy sử dụng regularization, việc này thường sẽ loại bỏ sự gia tăng phương sai.

Reply:
Tương tự, dấu in đậm "**" cần bám sát vào cụm bạn muốn in đậm. Ở đây vì có dấu cách nên phần sau của đoạn bị in đậm.

Reply:
Em sua het roi anh nhe.